src/load-styles.js

```diff
@@ -0,0 +1,62 @@
+import { CDN_HOST, getBody, getHeaders, getVersion, ONE_YEAR } from './common.js';
+
+const CSS = 'application/javascript';
+
+export async function handleLoadStylesRequest (httpApi) {
+
+  const versionsList = await httpApi.fetch(`versions-list.json`, CDN_HOST);
+  if (versionsList == null) {
+    return httpApi.sendResponse({ body: '', status: 500 });
+  }
+
+  const version = getVersion(versionsList, httpApi.getQueryParam('version'));
+
+  const depsManifest = await httpApi.fetch(`deps-manifest-${version.resolved}.json`, CDN_HOST);
+  if (depsManifest == null) {
+    return httpApi.sendResponse({ body: '', status: 500 });
+  }
+
+  const response = await getResponse(version, depsManifest, httpApi);
+
+  return httpApi.sendResponse(response);
+}
+
+export async function getResponse (version, depsManifest, httpApi) {
+
+  if (version.resolved == null) {
```

Comment:
this will never happen since the `httpApi.fetch(`deps-manifest-${version.resolved}.json`, CDN_HOST)` at line 14 may have failed.

I would have sent a 404 status code instead of 400. From my point of view, the query is valid but there is no resource for the given parameters.